internal/services/loadbalancer/backend_address_pool_resource_test.go

```diff
@@ -103,6 +103,50 @@ func TestAccBackendAddressPoolStandardSkuRequiresImport(t *testing.T) {
 	})
 }
 
+func TestAccBackendAddressPoolGatewaySkuBasic(t *testing.T) {
```

Comment:
we should have a _ in here
```suggestion
func TestAccBackendAddressPool_GatewaySkuBasic(t *testing.T) {
```